### PR TITLE
specify encoding for py3k stdout (utf-8)

### PR DESCRIPTION
**Pull request by msarahan**
Fixes errors raised during testing (sys.stdout.encoding is None for some reason)

### CODE REVIEW

**Comment by giampaolo** [comment]
I think it makes more sense to use sys.getfilesystemencoding() instead of "utf8". Can you please update your PR including changes to HISTORY and CREDITS files?

**Comment by msarahan** [comment]
Thanks for reviewing this.

A contributor to Conda, @jakirkham, came up with a better fix: https://github.com/conda/conda-build/pull/614

That style of fix is applied here, as well.  I have updated HISTORY and CREDITS for this PR.

File: test/test_psutil.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def sh(cmdline, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
     if stderr:
         warn(stderr)
     if PY3:
-        stdout = str(stdout, sys.stdout.encoding)
+        stdout = str(out, sys.stdout.encoding or sys.getfilesystemencoding() or 'utf-8')
```
Comment (jakirkham) [inline test/test_psutil.py]: Sorry, I took a look as I got pinged and was curious about where the problem arose here.

It looks like this change from `stdout` to `out` was unintentional.

Comment (msarahan) [inline test/test_psutil.py]: Thanks!  Good catch, clumsy cut/paste.

Comment (jakirkham) [inline test/test_psutil.py]: Glad to help. No worries. Happens to all of us.

**Pull request closed without merging**